plugins/Facebook/class.facebook.plugin.php

```diff
@@ -431,7 +431,7 @@ public function base_connectData_handler($sender, $args) {
         }
 
         // This isn't a trusted connection. Don't allow it to automatically connect a user account.
-        saveToConfig('Garden.Registration.AutoConnect', false, false);
+//        saveToConfig('Garden.Registration.AutoConnect', false, false);
```

Comment:
What was the reason for this for being disabled?
If the config is turned on for another plugin this may cause problems no?

Reply:
Was testing and it was annoying me, I meant to put it back.